improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -53,10 +53,12 @@ def set_up_height_cube(height_points, ascending=True):
     height_points = np.sort(height_points)
     cube = add_coordinate(
         cube, height_points, "height", coord_units="m")
+    cube.coord("height").attributes["positive"] = "up"
     cube.data = data.astype(np.float32)
 
     if not ascending:
         cube = sort_coord_in_cube(cube, "height", order="descending")
+        cube.coord("height").attributes["positive"] = "down"
```

Comment:
I've raised an issue with Fiona about our use of the "positive" attribute as I think we might have it wrong; your input might be useful. If so, we will come back and fix things here and elsewhere when we implement the fix everywhere.